build.gradle

```diff
@@ -608,6 +608,7 @@ project(':iceberg-delta-lake') {
     }
 
     task integrationTest(type: Test) {
+      useJUnitPlatform()
```

Comment:
The current master branch skips all the integration tests when running delta-lake CI. Seems we also need `useJUnitPlatform()` here when defining Junit5 integration test task.